src/app/config/no-config/no-config.component.ts

```diff
@@ -10,27 +12,40 @@ import { Config, ConfigService } from '../config.service';
   styleUrls: ['./no-config.component.scss'],
 })
 export class NoConfigComponent implements OnInit {
-  public page = 0;
-  public totalPages = 6;
+  public page: number = 0;
+  public totalPages: number = 7;
 
   public configUpdate: boolean;
   public config: Config;
   public configErrors: string[];
   public configValid: boolean;
   public configSaved: string;
 
-  public octoprintConnection: boolean;
+  public objectvalues = Object.values;
+  public octoprintNodes: any = {
+    'other': {
+      'display': 'Other (> 1.4.0)',
+      'name': 'other',
+      'version': '1.4.0',
+      'url': 'other',
+      'disable': false
+    }
+  };
+  public opInstance: any = this.octoprintNodes['other'];
+  public opApiMsg: string = '';
+  public manualEntry: boolean = true;
+  public octoprintConnection: boolean = false;
   public octoprintConnectionError: string;
 
-  public constructor(private configService: ConfigService, private http: HttpClient, private router: Router) {
+  public constructor(private configService: ConfigService, private http: HttpClient, private router: Router, private _electronService: ElectronService) {
     this.configUpdate = this.configService.isUpdate();
     console.log(this.configUpdate);
     if (this.configUpdate) {
       this.config = configService.getCurrentConfig();
     } else {
       this.config = {
         octoprint: {
-          url: 'http://localhost:5000/api/',
+          url: 'http://localhost:80/api/',
```

Comment:
We should stick with port 5000 here, since that is the official port from OctoPrint. (80 is just added from OctoPi)

Reply:
oh I didn't know that, maybe we could put a note, but since you official release used is octopi it's make more sense if it worked out of the box I know I got stuck on this a lot in the start.

Reply:
A note would be a good idea yeah. Although localhost:5000 should also work on OctoPi. OctoPi is just creating a reverse proxy from :80 to :5000 AFAIK.

Reply:
Odd it doesn't work on my Octodash install or my 2 other octopi installs, maybe I'm doing something wrong, I'll look into it.

Either way I'll add a note for it.